apps/editing-toolkit/editing-toolkit-plugin/block-inserter-modifications/contextual-tips.js

```diff
@@ -1,21 +1,11 @@
-import {
-	__unstableInserterMenuExtension,
-	__experimentalInserterMenuExtension,
-} from '@wordpress/block-editor';
+import { __unstableInserterMenuExtension as InserterMenuExtension } from '@wordpress/block-editor';
 import { useState } from '@wordpress/element';
 import { registerPlugin } from '@wordpress/plugins';
 import { debounce } from 'lodash';
 import ContextualTip from './contextual-tips/contextual-tip';
 
 import './contextual-tips/style.scss';
 
-// InserterMenuExtension has been made unstable in this PR: https://github.com/WordPress/gutenberg/pull/31417 / Gutenberg v10.7.0-rc.1.
-// We need to support both symbols until we're sure Gutenberg < v10.7.x is not used anymore in WPCOM.
-const InserterMenuExtension =
-	typeof __unstableInserterMenuExtension !== 'undefined'
-		? __unstableInserterMenuExtension
-		: __experimentalInserterMenuExtension;
```

Comment:
This was throwing errors because `__experimentalInserterMenuExtension` is no longer there. That made it easy enough to remove it and keep using only `__unstableInserterMenuExtension`.